Source/WebCore/dom/ContentVisibilityDocumentState.cpp

```diff
@@ -140,6 +142,7 @@ bool ContentVisibilityDocumentState::checkRelevancyOfContentVisibilityElement(El
     target.setContentRelevancy(newRelevancy);
     target.invalidateStyle();
     if (target.isConnected()) {
+        skippedContentStateDidChange(target, oldRelevancy->isEmpty(), !newRelevancy.isEmpty());
```

Comment:
Should this be outside of the `isConnected()` check? I don't really see why its should be inside.

Reply:
In KeyframeEffect::getKeyframes keyframes for disconnected elements are not included, so in theory this should be safe to do. @graouts WDYT?

Reply:
I don't feel strongly then.

Reply:
@graouts we will land as-is but still curious about your answer. If you think it is safe to move it into the "if (isConnected())" section, we can correct that later.